src/nspawn/nspawn-mount.c

```diff
@@ -815,6 +817,9 @@ static int mount_bind(const char *dest, CustomMount *m, uid_t uid_shift, uid_t u
 
                 if (chown(where, uid_shift, uid_shift) < 0)
                         return log_error_errno(errno, "Failed to chown %s: %m", where);
+
+                if (stat(where, &dest_st) < 0)
```

Comment:
This call is unneeded (The `dest_uid` is actually `uid_shift` :man_facepalming:).
Will fix.